contracts/sol6/mock/ReentrancyFeeClaimer.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity 0.6.6;
+
+import "../IKyberNetworkProxy.sol";
+import "../utils/Utils5.sol";
+import "../IKyberFeeHandler.sol";
+
+/// @dev contract to call trade when claimPlatformFee
+contract ReentrancyFeeClaimer is Utils5 {
+    IKyberNetworkProxy kyberProxy;
+    IKyberFeeHandler feeHandler;
+    IERC20 token;
+    uint256 amount;
+    uint256 count;
+
+    bool isReentrancy = true;
+
+    constructor(
+        IKyberNetworkProxy _kyberProxy,
+        IKyberFeeHandler _feeHandler,
+        IERC20 _token,
+        uint256 _amount
+    ) public {
+        kyberProxy = _kyberProxy;
+        feeHandler = _feeHandler;
+        token = _token;
+        amount = _amount;
+        require(_token.approve(address(_kyberProxy), _amount));
+    }
+
+    function claimPlatformFee() external {
+        count = 0;
+        feeHandler.claimPlatformFee(address(this));
+    }
+
+    function claimReserveRebate() external {
+        count = 0;
+        feeHandler.claimReserveRebate(address(this));
+    }
+
+    function setReentrancy(bool _isReentrancy) external {
+        isReentrancy = _isReentrancy;
+    }
+
+    receive() external payable {
+        if (count != 0) {
+            return;
+        }
+
+        if (!isReentrancy) {
+            return;
+        }
+
+        bytes memory hint;
+        kyberProxy.tradeWithHintAndFee(
```

Comment:
can use one of the simple swap API

Reply:
using `tradeWithHintAndFee` will updates the platformFee to this address